cmd/shield/commands/dispatcher.go

```diff
@@ -25,16 +25,18 @@ type HelpGroup struct {
 
 //Enumeration for HelpGroupTypes
 var (
-	InfoGroup     = &HelpGroup{name: "INFO"}
-	MiscGroup     = &HelpGroup{name: "MISCELLANEOUS"}
-	BackendsGroup = &HelpGroup{name: "BACKENDS"}
-	TargetsGroup  = &HelpGroup{name: "TARGETS"}
-	StoresGroup   = &HelpGroup{name: "STORES"}
-	PoliciesGroup = &HelpGroup{name: "POLICIES"}
-	JobsGroup     = &HelpGroup{name: "JOBS"}
-	ArchivesGroup = &HelpGroup{name: "ARCHIVES"}
-	AccessGroup   = &HelpGroup{name: "ACCESS"}
-	TasksGroup    = &HelpGroup{name: "TASKS"}
+	InfoGroup      = &HelpGroup{name: "INFO"}
+	MiscGroup      = &HelpGroup{name: "MISCELLANEOUS"}
+	BackendsGroup  = &HelpGroup{name: "BACKENDS"}
+	TargetsGroup   = &HelpGroup{name: "TARGETS"}
+	StoresGroup    = &HelpGroup{name: "STORES"}
+	SchedulesGroup = &HelpGroup{name: "SCHEDULES"}
```

Comment:
You should not be re-adding the SchedulesGroup -- maybe need a rebase against v8?